modules/channels.js

```diff
@@ -0,0 +1,32 @@
+var Channels = `
```

Comment:
Let's limit variable binding keywords to just `let` and `const`. In this case, `const` should do just fine.

Reply:
okay sir.